### ISSUE TITLE
Wrong budgeted count when BUDGET_ is in subtree

### ISSUE THREAD

**Issue by nico202** [Fuco1/org-clock-budget#13]
Hi, in my use case I need to track the total time spent working, and the time in some different project I'm working during my job.

This is a MWE
```org
* Job
:PROPERTIES:
:BUDGET_WEEK: 20:00
:END:
** Task 1
:PROPERTIES:
:BUDGET_WEEK: 10:00
:END:
** Task 2
:PROPERTIES:
:BUDGET_WEEK: 10:00
:END:
```

The total time for Job gets counted correctly, but the "Budgeted" clock should account only for the "Job" BUDGET, but it reports 40 (20 + 10 + 10).

What do you think?

Many thanks!

**Comment by Fuco1** [comment]
Yes you are right.  We should probably even show that if one task is a subtask of another, it should be nested under the parent, and since all the time bubbles up to the parent, only those "top level" tasks should be summed up in the bottom overview.  I had this problem before and solved it by moving around the budget settings but didn't fully realize what was going on.
